terraform/040-id-broker/vars.tf

```diff
@@ -581,13 +581,13 @@ variable "wildcard_cert_arn" {
   type = string
 }
 
-variable "app_id" {
+variable "appconfig_app_id" {
```

Comment:
I like this change, but do we need to consider this a breaking change due to the input variable renames?

Reply:
Good call. I added back the short versions with a "DEPRECATED" in the description.